.github/workflows/googlejavaformat-action.yml

```diff
@@ -1,16 +1,33 @@
 name: Format
 
-on:
-  push:
-    branches-ignore:
-      - master
+on: [push, pull_request]
```

Comment:
IMO executing the format action `on: pull_request` will not do anything useful. I tried using this extensively in #158 and it failed every time there were actual changes to format. The reason being that the runner within the pull request cannot see the git remote to push the new commit to. Where in an `on: push` action this is possible.

The example of the author of the action uses:
```
on: [push, pull_request]
```
in combination with:
```
with:
  args: "--set-exit-if-changed"
```
Setting the action up so new commits will never be created instead it will simply exit with an error code > 0 if formatting is missing. 

As we are actually formatting our commits (i.e. not setting `--set-exit-if-changed`) I think simply using `on: [push]` will be enough as every branch will have to be pushed to open a pull request.